dvc/lock.py

```diff
@@ -15,6 +16,12 @@
 
 DEFAULT_TIMEOUT = 5
 
+ERROR_MESSAGE = (
+    "cannot perform the cmd since DVC is busy and locked. You can delete "
+    ".dvc/lock to fix it, but first please make sure there's no other dvc "
```

Comment:
Use upper case name (DVC vs dvc)

Reply:
Hmmm. He uses "DVC" first, but the 2nd time I think he's referring to the `dvc` process. Should wrap it in back ticks ` though.

Reply:
I would rephrase it. The but-clause makes it harder to read than needed. Also "busy and locked" is too abstract, what does it mean? And cmd -> command.

So, it's like:

Cannot perform the command since other dvc process is running for this repo. If you are sure this is not the case, you might remove `.dvc/lock` to fix this ...

Then command line hints go. However, `ps xaf | grep dvc` doesn't make sense for Windows, unless again they don't use bash or somehting there.

Reply:
👍 , I agreed wtih @Suor regarding rephrasing. Also, including command line hints to check running processes I think is way too much. Let's just remove this part completely.